src/jit/codegenarm.cpp

```diff
@@ -1414,6 +1465,18 @@ instruction CodeGen::genGetInsForOper(genTreeOps oper, var_types type)
         case GT_ROR:
             ins = INS_ror;
             break;
+        case GT_ADD_LO:
+            ins = INS_add;
```

Comment:
AFAIK on ARM you need to specify that you want an instruction to set flags - it should be `ADDS` and not `ADD`. How/where is this handled by this code?